docs/cli/edgedb_database/edgedb_database_wipe.rst

```diff
@@ -0,0 +1,32 @@
+.. _ref_cli_edgedb_database_wipe:
+
+
+====================
+edgedb database wipe
+====================
+
+Destroy the contents of a :ref:`database <ref_datamodel_databases>`
+
+.. cli:synopsis::
+
+    edgedb database wipe [<options>]
+
+
+Description
+===========
+
+The database deleted will be one of these values: the value passed for the
+``--database``/``-d`` option, the value of ``EDGEDB_DATABASE``, or ``edgedb``.
+The contents of the database will be destroyed but the database itself will be
+preserved.
```

Comment:
The content is removed and the schema itself is also reset to the initial state before migrations. Seems somewhat like deleting and recreating a database with the same name, although I'm not 100% sure what other side effects there are.

Reply:
Added that the schema is also reset.